modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/AbstractCacheDumpTest.java

```diff
@@ -539,6 +539,11 @@ void createDump(IgniteEx ign, String name, @Nullable Collection<String> cacheGro
         ign.context().cache().context().snapshotMgr().createSnapshot(name, null, cacheGroupNames, false, onlyPrimary, true).get();
```

Comment:
Do we really need to keep method with old signature?

Reply:
No. Will remove.